test/functional/Java11andUp/src/org/openj9/test/condy/CondyGenerator.java

```diff
@@ -170,6 +171,47 @@
 			mv.visitEnd();
 		}
 
+		{
+			mv = cw.visitMethod( ACC_PUBLIC | ACC_STATIC, "triggerGC", "()I", null, null );
+			mv.visitCode();
+			mv.visitMethodInsn(INVOKESTATIC, "java/lang/System", "gc", "()V", false);
+			mv.visitMethodInsn(INVOKESTATIC, "java/lang/System", "gc", "()V", false);
+			mv.visitInsn( ICONST_1 );
+			mv.visitInsn( IRETURN );
+			mv.visitMaxs( 0, 0 );
+			mv.visitEnd();
+		}
+
+		{
+			mv = cw.visitMethod( ACC_PUBLIC | ACC_STATIC, "wrapper", "(II)V", null, null );
```

Comment:
Does this have to be generated code or can it live on GarbageCollectionCondyTest.java ?